### DISCUSSION TITLE
Unable to post message using kafka rest proxy

### DISCUSSION THREAD

**Discussion by rushikeshkulat** [strimzi/strimzi-kafka-operator#7680] [Q&A]
hi all,

im trying to POST/GET messages to/from kafka topic using kafka rest proxy endpoint. The kafka rest server is hosted on HPE node. The authentication method used is basic auth (it's taking mapr user and password). But i'm getting the below error- "

{

    "error_code": 80001,

    "message": "HPE Ezmeral Data Fabric Event Data Streams does not currently support this API. Please try to set streams.default.stream to return topics for default stream"

}"

my kafka-rest properties file looks like so-

listeners=http://ec2-16-170-16-252.eu-north-1.compute.amazonaws.com:8082/
authentication.enable=true
impersonation.enable=true
schema.registry.enable=false
#Security headers file
headers.file=/opt/mapr/kafka-rest/kafka-rest-6.0.0/config/headers.xml
host.name=hpezmzeralinstance0.localdomain

The error suggests to use the streams.default.stream parameter , but i'm unable to find the default stream name or default stream path. Please suggest how to update the parameter in properties file.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/117344115/203250874-a0bc9cf3-4d2f-4318-b0dd-fb019017d6ce.png)

**Comment by ppatierno** [comment]
Which rest proxy are you using, it doesn't seem to be the Strimzi one. Maybe the Confluent one? You should ask there in this case.

**Reply by rushikeshkulat** [reply]
this is Kafka REST Proxy for HPE Ezmeral Data Fabric Streams

**Reply by scholzj** [reply]
Sooo ... where does Strimzi fit in? Sorry, it is not entirely clear from your question.
